.kokoro/nightly/graalvm-native-17.cfg

```diff
@@ -30,4 +30,9 @@ env_vars: {
 env_vars: {
   key: "SECRET_MANAGER_KEYS"
   value: "java-it-service-account, java-bigqueryconnection-samples-secrets"
+}
+
+env_vars: {
+  key: "TEST_ALL_MODULES"
+  value: "true"
```

Comment:
I'm thinking to remove this env_var in a separate PR since we should even allow this flag to be toggled in the kokoro build. We can manually trigger the nightly build in fusion if we want to test all modules.

Reply:
this is the wrong file (should be presubmit)